cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -32,7 +32,8 @@ spec:
             - --sink=gcm
             - --sink=gcmautoscaling
             - --sink=gcl
-            - --sink_frequency=2m
+            - --stats_resolution=30s
+            - --sink_frequency=1m
```

Comment:
Did you run for a day with this config? In the past I have seen write failures after a few hours from GCM.